system/Config/Services.php

```diff
@@ -378,7 +407,7 @@ public static function logger(bool $getShared = true)
 			return static::getSharedInstance('logger');
 		}
 
-		return new \CodeIgniter\Log\Logger(new Logger());
+		return new \CodeIgniter\Log\Logger(new LoggerConfig());
```

Comment:
We should probably use `config('Logger')` here so that we take advantage of any caching and don't create any more instances than we need.

Reply:
ok